### ISSUE TITLE
suggestion: New mythological origins

### ISSUE THREAD

**Issue by norfaremc** [muon-rw/Medieval-Origins-Revival#29]
Hello there! I absolutely adore this mod. It's very well balanced and so much fun.

I was wondering if you have plans on making any more origins? I feel like, to complete this perfect medieval fantasy list, a few origins are missing. I've dabbled in origin making myself, and I know how complex it is, but I am not nearly as great at making origins as you guys are.

Here's some of my origin suggestions, if you do choose to make more origins in the future:
Minotaur
Gnome
Centaur
Nymph
Kobold
Hobbit
Angel (seraphim)
Ent

**Comment by muon-rw** [comment]
Some interesting ideas here! In general I don't want to add *too* many origins (the selection process is already a bit overwhelming) but I'll likely implement a few of these at some point in the future.

**Comment by norfaremc** [comment]
Ahaha, awesome! I know I sent a long list, totally get it. I don't expect any or all of them to be implemented, but some of these are just origins my players have requested! Most requested are Kobold, Nymph, Centaur and Minotaur.

**Comment by norfaremc** [comment]
If possible, I'd love if you could remake the original Merling origin. Right now, Sirens (like Valkyrie) are pretty much exclusive to females, which then leaves any masculine presenting players unable to choose a water-based origin comfortably. Although I can keep the original Merling, it drastically falls behind in abilities compared to the other medieval origins. I've even created a pixelated picture for a potential Merling remake, if you're willing to do it! If you were to create this origin, you can absolutely take this picture with no credit!! :3 Merrow or Merfolk would be great unisex names.

![Image](https://github.com/user-attachments/assets/42aaa149-5f46-46ca-b7f1-fe395a88f463)

**Comment by norfaremc** [comment]
P.S., I had to make the picture to complete my origin tree for my modpack so it'd match the origin portraits you've made.

![Image](https://github.com/user-attachments/assets/f9f6d32a-7c6e-402b-b08c-b0a14c89516e)

**Comment by norfaremc** [comment]
Additionally, is there any way for me to rename origins? I wanted to make them a bit more lively in my server to go along with my storyline if possible.

**Comment by muon-rw** [comment]
Yes, you should just be able to replace the origin names with a resource pack! See: https://github.com/muon-rw/Medieval-Origins-New/blob/1.20.1-multiloader/common/src/main/resources/assets/medievalorigins/lang/en_us.json (and with some of those renames you'd probably want to update descriptions - all also resourcepack-able)

The sprite you've made is great (though I'll probably work on rescaling it to 32x32 for consistency), I'll look into this. Thank you!

**Comment by muon-rw** [comment]
By the way, how you've described dark elves is exactly how the moon elf is designed gameplay wise, and the "lore" for them is currently some blend between DnD's ideas of moon elves and dark elves. 

I'm not too sure if there's much room here for something so similar, unless moon elves were retconned somewhat drastically (which could be interesting! An "ethereal arcane mage" archetype as opposed to the existing stealth-rogue design feels a bit better)

**Comment by norfaremc** [comment]
Love the idea of moon elves and dark elves being separate origins, tbh. That’s the vision I had for them, dark elves being more rogue-like whereas moon elves are more ethereal mages. Ultimately you do have your hands full and I’d be thankful if you did or didn’t get to my requests lately! Thanks for all the hard work! Also, I’ll absolutely make a resource pack, I have no clue why I never considered it an option before despite it being so obvious, lol.
